frontend/src/pages/ModelTester/Dashboard2/Dashboard2.jsx

```diff
@@ -10,18 +10,18 @@ const Dashboard2 = () => {
   const [currUser, setCurrUser] = useState("");
 
   const backgroundColours = [
-    "#00CC00", // Darker bright green 1
-    "#66CC66", // Darker bright green 2
-    "#66FF66", // Bright green 3 (unchanged)
-    "#99CC99", // Darker bright green 4
-    "#99FF99", // Bright green 5 (unchanged)
+    "#FF5733", // Vibrant Red
+    "#33FF57", // Vibrant Green
```

Comment:
these colors need to change (I assume the CSS will handle that?

Reply:
Yea, this is just placeholder, we need to revamp the entire theme and layout of this page